sdk/nodejs/src/cloudClient.ts

```diff
@@ -61,6 +67,7 @@ export class DVCCloudClient {
                 })
             })
             .catch((err: AxiosError) => {
+                throwIfUserError(err)
```

Comment:
do the errors being thrown here make sense to the user and indicate what they did wrong?

Reply:
hmm not really, it just throws "AxiosError: Request failed with status code 401"
I thought that was all we got back from the server but the response.message actually has useful stuff like "Error: Invalid SDK token provided" and "Error: Invalid User Data to process request" so i updated it to throw that